src/components/dialogs/parameters/sensi/sensitivity-analysis-parameters.jsx

```diff
@@ -454,6 +454,11 @@ export const SensitivityAnalysisParameters = ({ parametersBackend, setHaveDirtyF
         isSubmitAction,
         reset,
     ]);
+    useEffect(() => {
+        if (params) {
+            reset(fromSensitivityAnalysisParamsDataToFormValues(params));
+        }
+    }, [params, reset, fromSensitivityAnalysisParamsDataToFormValues]);
```

Comment:
in this file :
```
        setSensitivityAnalysisParameters(studyUuid, null)
            .then(() => {
            .then(() => {
                return getSensitivityAnalysisParameters(studyUuid)
                return getSensitivityAnalysisParameters(studyUuid)
                    .then((params) => {
                    .then((params) => {
                        setSensitivityAnalysisParams(params);
                        setSensitivityAnalysisParams(params);
                        updateParameters(params);
                        updateParameters(params);
                    })
                    })
                    .catch((error) => {
                    .catch((error) => {
                        snackError({
                        snackError({
                            messageTxt: error.message,
                            messageTxt: error.message,
                            headerId: 'paramsRetrievingError',
                            headerId: 'paramsRetrievingError',
                        });
                        });
                    });
                    });
            })
            })
            .catch((error) => {
            .catch((error) => {
                snackError({
                snackError({
                    messageTxt: error.message,
                    messageTxt: error.message,
                    headerId: 'paramsChangingError',
                    headerId: 'paramsChangingError',
                });
                });
            });
            });
 ```
 You don't need to get after set if notification trigger an update.
 You don't need a local `sensitivityAnalysisParams` if you already have a synchronysed one (`params`)
 
 This file needs to be clean

Reply:
Yes,no need for fetch. On the other hand for the change of **sensitivityAnalysisParams**, I prefer to do that in another US since this kind of refactoring is to be done for all parameters.